tests/integration/ceph_base_object_test.go

```diff
@@ -192,6 +194,12 @@ func assertObjectStoreDeletion(t *testing.T, k8sh *utils.K8sHelper, namespace, s
 	sleepTime := 3 * time.Second
 	for i = 0; i < retry; i++ {
 		storeStr, err := k8sh.GetResource("-n", namespace, "CephObjectStore", storeName, "-o", "json")
+		// if cephobjectstore is not found, just return the test
+		// no need to check deletion phases as it is already deleted
+		if err != nil && strings.Contains(storeStr, errors.NewNotFound(v1.Resource("cephobjectstores.ceph.rook.io"), storeName).ErrStatus.Message) {
+			return
+		}
+
```

Comment:
I am concerned that this could be flaky if the version of `kubectl` and the version of the `errors` library have slight changes to the `NotFound` definition. Is it possible to change the usage above from `k8sh.GetResource` to a `clientset.Rook()` so that we can check NotFound with less potential for errors to creep in in the future?

Reply:
Actually the above error(storeStr) is coming from k8s and the `errors.NewNotFound` is also a k8s api error, so I think they would be mutual, if anything changed too